app/models/product.rb

```diff
@@ -23,10 +23,20 @@ class Product < ApplicationRecord
   has_one :product_display_group_product
   has_one :product_display_group, through: :product_display_group_product
 
+  # Instrument specifc
+  has_one(
+    :alert,
+    dependent: :destroy,
+    class_name: "InstrumentAlert",
+    foreign_key: :instrument_id,
+    inverse_of: :instrument
+  )
+  has_many :current_offline_reservations, -> { current }, class_name: "OfflineReservation"
```

Comment:
Why did you have to move them to this model? While I don't think it's wrong, I find it a bit weird, so I'd like to understand the reason for the change

Reply:
I had to move them to be able to call `inlcudes(:alerts, :current_offline_reservation)` on a product relation